production/helm/loki/values.yaml

```diff
@@ -61,6 +61,9 @@ config:
 
 deploymentStrategy: RollingUpdate
 
+## Use a deployment (standalone mode) or use a statefulset (distributed mode) as Kubernetes workload
```

Comment:
why distributed mode ?

Reply:
I would expect if you could only ever have one of them (standalone) then you would want a statefulset to enforce it.